src/explorer.ts

```diff
@@ -416,23 +424,25 @@ export class KubernetesFileObject implements KubernetesObject {
 
 class HelmReleaseResource implements KubernetesObject {
     readonly id: string;
+    readonly status: string;
 
-    constructor(readonly name: string) {
-        this.id = "helmrelease:" + name;
+    constructor(readonly helmRelease: any) {
```

Comment:
I am trying to smother `any`s as close to birth as possible; can we pass in a name and status, or a typed interface, please?

Reply:
Fixing this.